examples/experimental/reference-grant.yaml

```diff
@@ -1,7 +1,7 @@
 #$ Used in:
 #$ - site-src/concepts/security-model.md
 #$ - site-src/blog/2021/introducing-v1alpha2.md
-apiVersion: gateway.networking.k8s.io/v1alpha2
+apiVersion: gateway.networking.k8s.io/v1beta1
```

Comment:
Needs to move to standard dir

Reply:
I've added this to `standard`; however, it's not clear if that means it should be removed from `experimental`. There's some overlap between the two. Thoughts?